src/main/resources/templates/springboot/performancetest/performance-test.scala.tmpl

```diff
@@ -1,17 +1,25 @@
 package ${packageName}
 
+import com.blackbaud.gatling.bbauth.BBAuthHelper
 import io.gatling.core.Predef.{exec, forever, scenario}
 import io.gatling.http.Predef.http
 import io.gatling.core.Predef._
+import io.gatling.core.structure.{ChainBuilder, ScenarioBuilder}
+import io.gatling.http.protocol.HttpProtocolBuilder
+import scala.concurrent.duration._
+import scala.language.postfixOps
 
 class ${className} extends Simulation {
 
```

Comment:
Why not use com.blackbaud.gatling.SimulationConfiguration for these properties?

Reply:
This is done